recipes/icepapcms/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "icepapcms" %}
+{% set version = "3.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/icepapcms-{{ version }}.tar.gz
+  sha256: 42aa39c238228ad6ef1f9f926315b95285a0186e89196cfbf7cb6896eeb39b0a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - icepapcms = icepapcms.__main__:main
+    - ipapconsole = icepapcms.gui.ipapconsole:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - configobj
+    - icepap >=3.6.1
+    - ipy >=0.62
+    - ldap3
+    - pyqt
```

Comment:
Should this be

`pyqt >=5`

Since the package depends on PyQt5

Reply:
Yes you are right, I will update. Thanks for noticing!